sale_service_project/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Service Project
+====================
+
+This module was written to extends the functionality of sale workflow
+and allow you to create projects and tasks with materials and works
+from sale lines.
+You can invoice from sale order or analytic lines.
+
+Configuration
+=============
+
+To use this module, you need to:
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a service product and check 'Create Task Automatically'.
+2. Now you can add task works and materials for this product.
```

Comment:
Now you will see two new columns, one per works of task and one for materials which will be used in the task.